content/blog/2019/announcing-1.1/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing Istio 1.1
+description: Istio 1.1 release announcement.
+publishdate: 2019-03-14
+attribution: The Istio Team
+weight: 77
+---
+We are pleased to announce the release of Istio 1.1.
+
```

Comment:
Let's insert some buttons here pointing directly to the 1.1 release notes and the 1.1 download page.

I'll add the needed support in the site infra to enable that.

Reply:
I think this is the only thing holding up the post.  I've got the CTA later on...are you sure we need these buttons?